crates/guest/src/guest.rs

```diff
@@ -51,6 +51,7 @@ where
 /// - Deserialize whatever bytes we can import from the host after calling the host function
 /// - Return a `Result` of the deserialized output type `O`
 #[inline(always)]
+#[allow(improper_ctypes_definitions)]
 pub fn host_call<I, O>(
     f: unsafe extern "C" fn(usize, usize) -> DoubleUSize,
```

Comment:
This gives a warning that u128 is not an ffi safe interface. Since our CI requires that clippy passes with no warnings, this became blocking.

Reply:
Yeah this is a funny one. It's helpful to allow 32-bit or 64-bit builds but it would be invalid with u128. I'm not aware of a type that include the first two but not the other. Reasonable to ignore I think